extra_docs/ums/README.md

```diff
@@ -106,11 +106,22 @@ const user = await ums.signUp({
 const user = await ums.getUser('Elon Musk'); 
 ```  
   
-### Change user password  
-```javascript  
-ums.changePassword('Elon@tesla.com', oldPassword, newPassword);  
-```  
-  
+### Change/reset user password
+User's password can be changed whether the current password is known by calling:
+```javascript
+ums.changePassword('Elon@tesla.com', currentPassword, newPassword);
+```
+\
+When the current password isn't known, you need to request a password reset:
+```javascript
+ums.requestResetPassword('Elon@tesla.com');
+```
+\
+The user should then receive an e-mail with instructions on how to reset. This e-mail should include a reset token, that should be used to request the password reset:
```

Comment:
Too many `should`'s. I would change it to:
`If provided email is correct and existed the user will receive an e-mail with instructions on how to reset a password. This e-mail will include a reset token, that should be used in password reset:`